homeassistant/components/qnap/__init__.py

```diff
@@ -1 +1,85 @@
 """The qnap component."""
+from datetime import timedelta
+import logging
+
+from qnapstats import QNAPStats
+
+from homeassistant.const import (
+    CONF_HOST,
+    CONF_PASSWORD,
+    CONF_PORT,
+    CONF_SSL,
+    CONF_USERNAME,
+    CONF_VERIFY_SSL,
+)
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator
+
+from .const import DEFAULT_PORT, DEFAULT_TIMEOUT, DOMAIN, PLATFORMS
+
+UPDATE_INTERVAL = timedelta(minutes=1)
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup(hass, config):
+    """Set up the qnap environment."""
+    hass.data.setdefault(DOMAIN, {})
+
+    return True
```

Comment:
Please remove this whole block.
You should instead move the `setdefault` to `async_setup_entry`.

Reply:
Done